modules/User Admin/user_manage_add.php

```diff
@@ -434,7 +434,9 @@
 
     $row = $form->addRow();
         $row->addLabel('studentID', __('Student ID'))->description(__('Must be unique if set.'));
```

Comment:
How come the description was removed in the application form, but not here? I assumed it was removed in application form as uniqueField() causes it to be added automatically. But perhaps not??